clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioParsers.scala

```diff
@@ -22,13 +22,19 @@ trait ClioParsers {
   implicit def enumEntryParser[T <: EnumEntry: Enum](
     implicit c: ClassTag[T]
   ): ArgParser[T] = {
-    ArgParser.instance[T]("entry") { entry =>
-      val value = implicitly[Enum[T]].withNameOption(entry)
-      Either.cond(
-        value.isDefined,
-        value.get,
-        s"Unknown enum value $entry for type ${c.runtimeClass.getName} "
-      )
+    val enum = implicitly[Enum[T]]
+    val enumName = c.runtimeClass.getName
```

Comment:
Can't you get this value from the `enum` val?

Reply:
I don't think so. `enum` knows about its possible values but doesn't know its own class name.

Reply:
So something like `enum.getClass.getSampleName` wouldn't work?

Reply:
Ah, not sure! Will try it out.

Reply:
Tried it, and it appends a '$' to the end of the class name.

Reply:
:(